.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Summary
+
+Add a short summary of what this Pull Request does in 1-3 sentences in the first paragraph. 
+
+_If you are simply editing or adding a new article, this is all you need - feel free to omit the rest of this template entirely. For more technical Pull Requests, continue with the following:_
+
+### Changes
+
+- Provide a bulleted list of changes that are introduced by this Pull Request.
```

Comment:
One thing I have used elsewhere is to provide a number of explicit steps here, such as updating a [changelog] or passing the document through some formatting and proof reading processing tools. Considering the note you had on the [TOC] ticket it would likely be useful to include that as well. 

### Future Enhancement Idea
We may also want to add a few utilities/validation scripts to help smooth the PR flow. One such tool would be to ensure that a given addition is included in the page tree. 


[changelog]: http://keepachangelog.com/en/1.0.0/
[toc]: https://github.com/AdamsLair/duality-docs/issues/16

Reply:
Being a bit more explicit and providing a list of steps / points to check sounds like a good idea 👍 Will look into this later.

Validation scripts / tools are a good idea too, but I think we should move this into a separate issue.